lib/widgets/PaddedButton.dart

```diff
@@ -30,10 +31,11 @@ class PaddedButton extends StatelessWidget {
     final theme = Theme.of(context);
 
     Widget primaryButton = ElevatedButton(
-      style: ElevatedButton.styleFrom(
+      style: theme.elevatedButtonTheme.style?.merge(ElevatedButton.styleFrom(
```

Comment:
instead add this to the light and dark theme
    `ElevatedButton.styleFrom(
        foregroundColor: Colors.white, // changes the color of the text`